modules/references/client/components/create-reference/Experience.js

```diff
@@ -17,69 +18,80 @@ export default function Recommend({
   const { t } = useTranslation('references');
 
   const recommendQuestions = {
-    hostedMe: t('Would you recommend others to stay with them?'),
-    hostedThem: t('Would you recommend others to host them?'),
-    met: t('Would you recommend others to meet them?'),
+    hostedMe: t(
+      `Did you feel comfortable being hosted by ${referencedUserName}?`,
+    ),
+    hostedThem: t(`Did you feel comfortable hosting ${referencedUserName}?`),
+    met: t(`Did you feel comfortable meeting ${referencedUserName}?`),
```

Comment:
I suggest we remove the username for reasons described in https://github.com/Trustroots/trustroots/pull/1389/files#r411222341